src/legacy/ui/ui_render/ui_render_mixin.js

```diff
@@ -164,12 +164,11 @@ export function uiRenderMixin(kbnServer, server, config) {
   });
 
   server.route({
-    path: '/app/{id}',
+    path: '/app/{id}/{path?}',
```

Comment:
does anyone read this parameter?

Reply:
No, but it's necessary for Hapi to match SPA routes like `/app/foo/deep/path`

Reply:
I see. optional: for consistency with other code `'/app/{id}/{p*}',
[1](https://github.com/restrry/kibana/blob/fe51d3419fefc464e8a5672d66bb625d7d5e6b06/src/legacy/server/http/index.js#L98)
[2](https://github.com/restrry/kibana/blob/fe51d3419fefc464e8a5672d66bb625d7d5e6b06/x-pack/legacy/plugins/code/server/routes/redirect.ts#L29)
[3](https://github.com/restrry/kibana/blob/fe51d3419fefc464e8a5672d66bb625d7d5e6b06/x-pack/legacy/plugins/reporting/server/routes/generate.ts#L45)